src/ucs/sys/sys.c

```diff
@@ -1479,12 +1484,14 @@ ucs_status_t ucs_sys_readdir(const char *path, ucs_sys_readdir_cb_t cb, void *ct
     return res;
 }
 
-static ucs_status_t ucs_sys_enum_threads_cb(struct dirent *entry, void *_ctx)
+static ucs_status_t
+ucs_sys_enum_threads_cb(const struct dirent *entry, void *_ctx)
```

Comment:
seems unrelated change

Reply:
Following this [comment](https://github.com/openucx/ucx/pull/8698#discussion_r1146131684)

If I'm not mistaken adding const will require changing the cb signature.